src/it/java/com/ahaidychuk/dependencyversionupdater/repository/ProjectRepositoryIT.java

```diff
@@ -0,0 +1,50 @@
+package com.ahaidychuk.dependencyversionupdater.repository;
+
+import com.ahaidychuk.dependencyversionupdater.entity.Project;
+import com.github.database.rider.core.api.configuration.DBUnit;
+import com.github.database.rider.core.api.dataset.DataSet;
+import com.github.database.rider.spring.api.DBRider;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import java.util.stream.Collectors;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.hasSize;
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@DBRider
+@DBUnit(caseSensitiveTableNames = true)
+class ProjectRepositoryIT extends DBTestContainer {
+
+  @Autowired
+  private ProjectRepository projectRepository;
+
+  @Test
+  @DataSet("project_insert.yml")
+  void testFindAllActive() {
+    final var projects = projectRepository.findAllByActiveTrue();
+
+    assertThat(projects, hasSize(2));
+    assertThat(
+            projects.stream().map(Project::getProjectName).collect(Collectors.toList()),
```

Comment:
Minor
```suggestion
            projects.stream().map(Project::getProjectName),
```

or you can do:
```
assertThat(projects).extracting(Project::getProjectName).containsInAnyOrder(....)
```

Reply:
Updated and replaced Hamcrest with AssertJ for all tests` assertion